update.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+if [[ -z "$1" ]]; then
+    echo "Usage: bash update.sh version" 1>&2
+    echo ""
+    echo "Example: bash update.sh v0.34.0" 1>&2
+    exit 1
+fi
+
+VERSION=$1
+
+pushd third_party/substrait/
```

Comment:
This assumes that the working directory is the same as the script file location. You can make calling this script from any location work by adding:
```suggestion
DIR=$(cd $(dirname $0) && pwd)
pushd "${DIR}"/third_party/substrait/
```

Reply:
Nice improvement, thanks @mbrobbel !